### DISCUSSION TITLE
역 별 주로 내리는 승객 특징 도움 요청

### DISCUSSION THREAD

**Discussion by Dukkov** [SesacWannaSIT/WannaSIT#37] [General]
착석꿀팁에 사용할 json 파일을 수정하다가 승객 특징은 도저히 떠오르는게 없어서 일단 지웠습니다.

passengerInfo 유형은 다음과 같습니다.
1. 공무원
2. 관광객
3. 직장인
4. 인근 주민
5. 젊은 사람
6. 대학생
7. 쇼핑객
8. 스포츠 관람객
9. 다양한 사람

passengerInfo 유형에 따라 승객 특징을 작성하고 DB에 업데이트 할 예정입니다.

각 승객 유형에 매칭되는 승객 특징 아이디어 제공해주시면 반영하겠습니다. 코멘트 달아주세요!

[착석꿀팁 json 파일](https://github.com/SesacWannaSIT/WannaSIT/files/13857284/stationTips.json)

**Comment by 0unk0** [comment]
공무원은 좀 아닌거같아요...(from.정미^^)

**Comment by 0unk0** [comment]
다양한 사람은 뭐냐?ㅋㅋㅋ (from.정미^^)

**Comment by 0unk0** [comment]
stationInfo(ex. 회사 밀집 지역) 유지
passengerInfo(ex. 다양한 사람, 인근 주민 유형 X -> 자세한 특징)

**Comment by Dukkov** [comment]
[221~233 착석꿀팁](https://github.com/SesacWannaSIT/WannaSIT/files/14103229/stationTips.json)

**Comment by WhyBusyy** [comment]
[stationTips.json](https://github.com/SesacWannaSIT/WannaSIT/files/14113521/stationTips.json)

일단 데이터는 이정도 수정,, stationInfo도 여러 개 수정해서 6개 정도로 분류해서 이미지 찾아볼게!
